plugins/09_Shadowlands/localization/enUS.lua

```diff
@@ -889,6 +1035,9 @@ L['coreless_automa_warning'] = '|cFFFF8C00Wait for the automa to actually start
 
 L['olea_manu'] = 'Sells collectibles and recipes in exchange for {currency:1979}.'
 
+L['venaris_fate_sublabel'] = 'Located in the |cFFFFFF00Creation Catalyst|r'
+L['venaris_fate_note'] = 'Interact with the corpse of {npc:185083}. 5 days later you will receive a letter in the mail from {npc:185083} along with the {item:192485}'
```

Comment:
This note needs more information. It should mention:

1. That you must speak with Ven'ari in the Maw first before her corpse will appear at the Creation Catalyst.
2. If she is not incorporeal, additional story progress of some form is required.

![image](https://user-images.githubusercontent.com/64381982/174492360-0182daef-3796-4c03-862d-bc8bf0c7c1c2.png)

The quest ID that flips once you have spoken to her in this form is `65470`.